### PR TITLE
Create Event Class

### PR DESCRIPTION
**Pull request by hieudan225**
Create Event Class

### CODE REVIEW

File: step-capstone/src/App.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ class App extends React.Component {
   render () {
     return(
       <div className="App">
-        <TravelObject type="flight" />
+        <TravelObject type="event" data={data} />
       </div>
     );
   }
```
Comment (zirving) [inline step-capstone/src/App.js:12]: could you pull this branch down and git checkout this file? Its gonna definitely cause a merge conflict with App and doesnt need to be committed

File: step-capstone/src/components/Event.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          type: "event",
+          title: "Visiting Sherlock",
+          location: "221b Baker Street, London",
+          timeStart: "8:30",
+          timeEnd: "10:30",
+          description: "Additional notes"
+        }
+
+*/
+export default function Flight( data ) {
```
Comment (zirving) [inline step-capstone/src/components/Event.js:17]: change Flight to Event

File: step-capstone/src/components/Event.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    return(
+        <Grid item>
+            <Typography variant="h6" gutterBottom>Title { data.title } to { data.title }</Typography>
+            <Typography variant="subtitle2" gutterBottom>Location: { data.location }</Typography>
+            <Typography variant="subtitle2" gutterBottom>Time start: { data.timeStart }</Typography>
+            <Typography variant="subtitle2" gutterBottom>Time end: { data.timeEnd }</Typography>
```
Comment (zirving) [inline step-capstone/src/components/Event.js:23]: I dont think we want all the descriptions of what each thing is to appear. so it should just be {data.location} for example, not Location: {data.location}. Also, I think Title is still left in a data format, and Time Start and Time End, if we do display them, should be done like start - end

File: step-capstone/src/components/Event.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import { Box, Grid, Typography } from '@material-ui/core'
+
+/*
+Takes data in this form:
+        {
+          finalized: true,
+          type: "event",
+          title: "Visiting Sherlock",
+          location: "221b Baker Street, London",
+          timeStart: "8:30",
```
Comment (ganewman42) [inline step-capstone/src/components/Event.js:11]: Timezones would be a good thing to add here, including display handling for them.

File: step-capstone/src/components/Event.js (hunk 3 above)
Comment (ganewman42) [reply step-capstone/src/components/Event.js:23]: I agree with not labeling title and location, but labeling start and end times makes an equal amount of sense to me as start-end if you prefer that.

Comment (emaohn) [reply step-capstone/src/components/Event.js:23]: I believe the date/time will come in as a Data object, so you would need to convert that to a string format (toString())

**Pull request closed without merging**